manifests/components/nginx-ingress.jsonnet

```diff
@@ -141,11 +141,23 @@ local NGNIX_INGRESS_IMAGE = (import "images.json")["nginx-ingress-controller"];
 
   hpa: kube.HorizontalPodAutoscaler($.p + "nginx-ingress-controller") + $.metadata {
     target: $.controller,
-    spec+: {maxReplicas: 10},
+    spec+: {
+      // Put a cap on growth due to (eg) DoS attacks.
+      // Large sites will want to increase this to cover legitimate demand.
+      maxReplicas: 10,
```

Comment:
Do we need a minReplicas: 2 or some such to prevent auto-scaling from messing things up?

Reply:
`kube.libsonnet` already inserts a `minReplicas=$.target.spec.replicas` for us.